allensdk/brain_observatory/ecephys/nwb/__init__.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import pynwb
+
+
+namespace_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "AIBS_ecephys_namespace.yaml"))
```

Comment:
Maybe worth using pathlib?

`namespace_path = Path(__file__).resolve().parent / "AIBS_ecephys_namespace.yml"`
or is this python 2 stuff?

Reply:
not python 2, but pynwb chokes on Path objects here.